scripts/database-management/DMFlyway.ps1

```diff
@@ -129,11 +130,11 @@ function Invoke-DatabaseBuild {
             $versions = Get-Version -sqlFolder $baselineSnapshotFolder
             $baselineVersion = ($versions | Measure-Object -Maximum).Maximum
             Invoke-FlywayBaseline -hostName $hostName -port $port -dbName $dbName -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion
-            Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $baselineSnapshotFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion
+            Invoke-Flyway -hostName $hostName -port $port -dbName $dbName -scriptFolder $baselineSnapshotFolder -MigrationHistoryTable "MigrationHistory" -projectRoot $projectRoot -baselineVersion $baselineVersion -validateMigrations:$validateMigrations
```

Comment:
This line is long and could use a manual wrap.

Reply:
You're right, I'll do that for this and the other line below this.